src/views/TagsInput.jsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import { WithContext as ReactTags } from 'react-tag-input';
+
+const KeyCodes = {
+  comma: 188,
+  enter: 13,
+};
+
+const delimiters = [KeyCodes.comma, KeyCodes.enter];
+
+class TagsInput extends React.Component {
+  state = {
+    tags: [
+      { id: 'Business', text: 'Business' },
```

Comment:
#### Nitpick
- Whats going on here? is this hard coded?

Reply:
This is here to promote a good user experience with the tag input. It is a sample. A user has the option of removing them or adding more tags. It is deactivated by default and not sent as a part of the request body until the user adds a new tag or removes one of the sample tags.